release/gff/gff.am.gff_amharic/source/gff.am.gff_amharic.model.ts

```diff
@@ -0,0 +1,14 @@
+/*
+  GFF Amharic 1.0 generated from template.
+  
+  This is a minimal lexical model source that uses a tab delimited wordlist.
+  See documentation online at https://help.keyman.com/developer/ for
+  additional parameters.
+*/
+
+const source: LexicalModelSource = {
+  format: 'trie-1.0',
+  wordBreaker: 'default',
+  sources: ['am-unilex.txt'],

```

Comment:
One minor thing and then the PR is ready.

Could you rename the wordlist to `am-unilex.tsv` (tab-separated values)?